theme-settings.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @file
+ * theme-settings.php
+ */
+
+/**
+ * Implements hook_form_system_theme_settings_alter().
+ */
+function europa_form_system_theme_settings_alter(&$form, &$form_state) {
+  // Build the form.
+  if (empty($form['europa'])) {
```

Comment:
I don't understand the need of adding the fieldset, because I think we always have it no?